src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -30,6 +30,8 @@ private void applyContaminationFilter(final M2FiltersArgumentCollection MTFAC, f
         final Genotype tumorGenotype = vc.getGenotype(tumorSample);
         final double[] alleleFractions = GATKProtectedVariantContextUtils.getAttributeAsDoubleArray(tumorGenotype, VCFConstants.ALLELE_FREQUENCY_KEY,
                 () -> new double[] {1.0}, 1.0);
+        final int maxIndex = MathUtils.maxElementIndex(alleleFractions);
```

Comment:
You probably didn't mean to include this

Reply:
whoops, slipped in from that other branch